docs/installation.rst

```diff
@@ -66,10 +66,25 @@ To download the bot you can either:
         WantedBy=multi-user.target
 
     Credit to GitHub user utdrmac.
+    
+    
+    The permission on the unit file needs to be set to 644 :
```

Comment:
Since this is only necessary on some OSs, please rephrase this to say "may" need to be set...
Very useful Troubleshooting tip! :)